zcash_proofs/src/lib.rs

```diff
@@ -33,20 +33,48 @@ pub mod sprout;
 )]
 pub mod prover;
 
```

Comment:
Move the imports of `std::fs::File` and `std::io::BufReader` on lines 14-15 into the functions that use them, so that they're not unused imports when "download-params" and "local-prover" features are not enabled.

Reply:
Some functions were being compiled conditionally, but they should have always been available.

Fixed in 40d9115cc51bdfe4f55c986c30216ce65e8410ea